src/scythe-bidder/combination-selector.tsx

```diff
@@ -0,0 +1,266 @@
+/** @jsx jsx */
+
+import React from "react";
+import { jsx } from "@emotion/core";
+import {
+  Faction,
+  FactionMatCombinations,
+  FMComboAction,
+  FMComboActionType,
+  Mat,
+} from "./types";
+import { Modal, Table, Checkbox, Button, Select, Tooltip } from "antd";
+import { MATS_IFA, FACTIONS_IFA, PRESETS, BANNED_COMBOS } from "./constants";
+import { getFactionIcon } from "./utils";
+import cloneDeep from "lodash/cloneDeep";
+import { isEqual } from "lodash";
+import { WarningFilled } from "@ant-design/icons";
+
+const { Column } = Table;
+
+function getData(value: FactionMatCombinations) {
+  return FACTIONS_IFA.map((faction) => {
+    const result = MATS_IFA.reduce(
+      (acc, curr) => {
+        return { ...acc, [curr]: value[faction][curr] };
+      },
+      { imageUrl: getFactionIcon(faction), name: faction }
+    );
+    return result;
+  });
+}
+
+function CombinationSelector({
+  value,
+  onToggle,
+  onFactionToggle,
+  onMatToggle,
+  onPresetSelect,
+}: {
+  value: FactionMatCombinations;
+  onToggle: (f: Faction, m: Mat, v: boolean) => void;
+  onFactionToggle: (f: Faction, v: boolean) => void;
+  onMatToggle: (m: Mat, v: boolean) => void;
+  onPresetSelect: (value: FactionMatCombinations) => void;
+}) {
+  return (
+    <div css={{ display: "flex", gap: 12 }}>
```

Comment:
Currently has some visual bugs on mobile. Adding `overflow-x: scroll` to the css on line 47 should fix this.

Reply:
On second thought, probably `overflow-x: auto`, no need to see that scrollbar when it is not needed 😅

Reply:
Great catch, thank you!